src/relay/transforms/partition_graph.cc

```diff
@@ -301,7 +302,7 @@ class Partitioner : public MixedModeMutator {
     }
 
     std::string target = end_node->attrs.as<CompilerAttrs>()->compiler;
-    std::string name = target + "_" + std::to_string(region->GetID());
+    std::string name = target + "_" + std::to_string(region->GetID()) + region->GetName();
```

Comment:
Let's make it clearer: target - region (function) - index.

```suggestion
    std::string name = target + "_" + region->GetName() + "_" + std::to_string(region->GetID());
```

Reply:
Agree, the only reason I put in that way is due to when name is "dafault" then GetName() method will return an empty string and I was afraid it would cause some issues if i changed that. I agree this is more clear and will test it.

Reply:
Agree, the only reason I put in that way is due to when name is "dafault" then GetName() method will return an empty string and I was afraid it would cause some issues if i changed that. I agree this is more clear and will test it.